tests/index.test.ts

```diff
@@ -483,3 +481,14 @@ async function createReportFileIfNotExists() {
 
   return promise;
 }
+
+describe("ActionInputs validation", () => {
+  it("accepts valid severities", () => {
+    const inputs = { sysdigSecureToken: "t", imageTag: "foo", severityAtLeast: "critical", mode: "vm" };
+    expect(() => ActionInputs.overridingParsedActionInputs(inputs)).not.toThrow();
+  });
+  it("rejects invalid severityAtLeast", () => {
+    const inputs = { sysdigSecureToken: "t", imageTag: "foo", severityAtLeast: "severe", mode: "vm" };
+    expect(() => ActionInputs.overridingParsedActionInputs(inputs)).toThrow(/Invalid severity-at-least/);
```

Comment:
Nitpick, we could ensure that we are also showing the expected valid values. Not to change now, not really that important.